src/bssl.rs

```diff
@@ -38,6 +38,7 @@ macro_rules! bssl_test {
             }
 
             init::init_once();
+            ::std::env::set_current_dir(::test::ring_src_path()).unwrap();
```

Comment:
On a non-jailbroken iOS, the only (known) way to start a process is to make it an app, and lauching it as an app. It will only be started with the working directory set to `/`, I could not find a way around that.

So we have two ways to solve this: using absolute paths (as I did for the pure Rust tests) or cd-ing to the right place just before running tests and keep relative paths. On principle, the first approach feels better as you're not altering what is basically a process-wide global variable. I used the cd strategy for the bn tests because I felt the required changes would be too invasive (as we need to pass the path through the rust-to-c test machinery), but if you feel otherwise, I can give it a shot.